### PR TITLE
add all projects statistics

### PR DESCRIPTION
**Pull request by gcor**

### CODE REVIEW

File: components/Project/Project.module.css [hunk 1]
```diff
@@ -1,190 +1,190 @@
 .project {
-    padding: 40px 44px 44px 36px;
-    vertical-align: top;
-    box-shadow: 0 3px 12px -2px rgba(17, 12, 46, .15);
-    width: 100%;
-    break-inside: avoid;
-    background: white;
-    border-radius: 8px;
-    transition: .1s ease box-shadow;
+  padding: 40px 30px;
```
Comment (sashachabin) [inline components/Project/Project.module.css]: nit
`40px 32px`

we haven't approved this practice yet
but we try to use multiples of 8 (or 4?)

File: components/Activity/Activity.jsx [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+
+  if (!activity.length) {
+    return <Loading />;
+  }
+
+  return (
+    <div className={styles.graph}>
+      <ul className={styles.months}>
+        {months.map((m) => (
+          <li key={m}>{m}</li>
```
Comment (sashachabin) [inline components/Activity/Activity.jsx:95]: Let's add tooltip with date and number contributions on hover :)

image

File: components/Activity/Activity.jsx [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+    };
+  }, [setLimit]);
+
+  if (!activity.length) {
+    return <Loading />;
+  }
+
+  return (
+    <div className={styles.graph}>
+      <ul className={styles.months}>
```
Comment (sashachabin) [inline components/Activity/Activity.jsx:93]: Let's add the `aria-hidden="true"` and hide a lot of useless information for the screen readers

File: components/Activity/Activity.jsx [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+      window.removeEventListener("resize", onResize);
+    };
+  }, [setLimit]);
+
+  if (!activity.length) {
+    return <Loading />;
+  }
+
+  return (
+    <div className={styles.graph}>
```
Comment (sashachabin) [inline components/Activity/Activity.jsx]: Let's add `N contributions in {pediod}`

image

File: components/Activity/Activity.jsx (hunk 2 above)
Comment (sashachabin) [reply components/Activity/Activity.jsx:95]: We can use a simple CSS pseudo-element tooltip for now 😏
https://codepen.io/esedic/pen/NWWQbZG

File: styles/globals.css [hunk 5]
```diff
@@ -2,8 +2,8 @@ html,
 body {
   padding: 0;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
-    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+  font-family: "Iset Sans", -apple-system, BlinkMacSystemFont, Segoe UI, Roboto,
```
Comment (sashachabin) [inline styles/globals.css:5]: Let's add for perfect font rendering
```css
    -webkit-font-smoothing: antialiased;
    -moz-osx-font-smoothing: grayscale;
```

File: components/ProjectList/ProjectList.jsx [hunk 6]
```diff
@@ -1,19 +1,95 @@
...
+import { useEffect, useState } from "react";
+import { Activity } from "../Activity/Activity";
+import { Contributors } from "../Contributors/Contributors";
+import Project from "../Project/Project";
+import styles from "./ProjectList.module.css";
 
 const cx = classNames.bind(styles);
 
-/** @param {{ projectsData: import("../../types/projects-config").Project[] }} */
 export default function ProjectList({ projectsData }) {
```
Comment (sashachabin) [inline components/ProjectList/ProjectList.jsx]: The idea of decomposition.

It seems that `ProjectList` is a separate component with a list of repositories.
Maybe we'll make some `<Dashboard>` that would consist of heading and 3 areas with:
   - Activity
   - Pulse
   - ProjectList

File: components/Project/Project.module.css (hunk 1 above)
Comment (gcor) [reply components/Project/Project.module.css]: done

File: components/Activity/Activity.jsx (hunk 2 above)
Comment (gcor) [reply components/Activity/Activity.jsx:95]: added

File: components/Activity/Activity.jsx (hunk 3 above)
Comment (gcor) [reply components/Activity/Activity.jsx:93]: added

File: components/Activity/Activity.jsx (hunk 4 above)
Comment (gcor) [reply components/Activity/Activity.jsx]: added for "last year"

File: styles/globals.css (hunk 5 above)
Comment (gcor) [reply styles/globals.css:5]: added

File: components/ProjectList/ProjectList.jsx (hunk 6 above)
Comment (gcor) [reply components/ProjectList/ProjectList.jsx]: We already have similar structure. Renamed ProjectList to Dashboard and added the structure: Dashboard -> that consists of Activity, Contributors, Projects

**Comment by sashachabin** [comment]
Thanks!

**Pull request merged**